test/specs/submission.spec.js

```diff
@@ -5,10 +5,12 @@ var Sequelize = require('sequelize');
 var sequelize = new Sequelize(config.db);
 var Application = sequelize.import("../../server/models/application");
 var SubmissionAttempts = sequelize.import("../../server/models/submissionAttempts");
-var queueLocation = config.rabbitMQ.queueLocation;
+var queueLocation = ""; // config.rabbitMQ.queueLocation;
 var cp = require('child_process');
 var appId = '2482';
-describe('Work with Successful Submissions', function () {
+
+// Integration tests that rely on rabbitMQ which is no longer used
+describe.skip('Work with Successful Submissions', function () {
```

Comment:
I know we had a chat about this on Slack already but thought I'd answer it here as well. There's a preference from Kainos to skip the tests instead of deleting them that's why it exists.

They'll be maintaining it so I'm happy to listen to their request.